.github/workflows/publish.yaml

```diff
@@ -42,6 +42,16 @@ jobs:
 
       - run: bun run build
 
+      - name: Prepare README for npm
+        run: |
+          # Create a new README with the modified content
+          {
+              head -n 2 README.md
+              echo "![zondax](docs/zondax_light.png)"
+              tail -n +4 README.md
+          } > README.md.tmp
+          mv README.md.tmp README.md
+
```

Comment:
We should leave the black-filled zondax image
You will need to modify the package.json to include only the base readme.md file (double check this)